### ISSUE TITLE
Change of tag line

### ISSUE THREAD

**Issue by SimonBiggs** [pymedphys/pymedphys#1568]
I propose we change the following:

> A community effort to develop an open standard library for Medical Physics in Python. We build high quality, transparent software together via peer review and open source distribution. Open code is better science.

to:

> A community effort to develop an open standard library for Medical Physics in Python. Helping Medical Physicists verify radiotherapy cancer treatments are safe and effective in their day-to-day job while supporting Medical Physics cancer research with practical tools.

What are other's thoughts?

**Comment by sjswerdloff** [comment]
I would not limit it to oncology.  Diagnostics is a significant part of medical physics, and keeping science as part of the foundation is important.
My involvement was focused on enabling research, even if that research was oncology.

**Comment by SimonBiggs** [comment]
Yup, makes sense. Do you have an alternative wording? Certainly happy to have the research component be first and foremost.

**Comment by pchlap** [comment]
I'd agree with @sjswerdloff that I am contributing to this more from the research side. I quite like the "Open code is better science"

If you want to get in something about helping medical physicists in their day-to-day tasks, what about simply extending the original tagline to something like:

### A community effort to develop an open standard library for Medical Physics in Python. We build high quality, transparent software together via peer review and open source distribution. Open code is better science. Peer reviewed code ensures safe and effective development of tools in a Medical Physicist's day-to-day job.

...or something along those lines :)

**Comment by SimonBiggs** [comment]
I certainly also quite like the "open code is better science".

I guess what I've realised is when a non-medical physisict or
non-radiotherapy domain person reads the current tag line, they don't
actually come away knowing what PyMedPhys is.

My concern with just extending it is it is not really a tagline anymore. A
sentence or two that clearly explains what it is and why it is
valuable/needed. Something that can be used to describe the project at a
conference, or on a grant application. However, most of the points you've
flagged might be able to be in there... We just likely need to be
significantly more efficient with our wording.

We can move key concepts that we don't want to lose into an "about
PyMedPhys" paragraph that belongs right at the top of the readme... But a
tagline should really be a sentence (or two, maybe).

**Comment by sjswerdloff** [comment]
How about:
**A community effort to develop an open standard library for Medical Physics in Python. 
Helping Medical Physicists in their efforts to verify safe and effective treatment or diagnosis as part of their daily work, and also supporting research into cancer and other diseases with practical analysis tools of Medical Physics.**

After that you can talk about the value of open code if you think there is room for more words.
But the first sentence puts a stake in it when it mentions community and open.  One can't achieve that with closed source.  The second sentence can be achieved with closed source, so it's not the same as the first sentence, but it also addresses the "why would anyone care", and in particular "anyone who doesn't already know what a Medical Physicist does" (which is most of the planet).  I'd leave the sentence about community and open as the first one because otherwise we wouldn't all be here.  

I agree that open code is better science, and it belongs somewhere, I'm just trying to be as concise as possible.
I disagree that peer reviewed code ensures safe and effective development.  It certainly contributes to it, but I would argue that depending on the definition of peer, it may not be necessary, and it certainly is not sufficient.  Don't get me wrong, I think there is huge value in peer review.  Simon has been incredibly disciplined in the approach to development and testing in the project, and that's more than just "open + peer review".  The CI/CD approach and the enforcement of coding guidelines including unit tests makes a huge difference. 

But it is possible for a lone physicist in a group of software engineers,  product owner(s), and quality engineers who are following a carefully defined process including version control, to perform safe development (which I interpret as meaning that the process of developing the software causes no harm, and the resulting software causes no harm).  If you defined the software engineers as peers of the physicist, for the purposes of code review, the confirmation of the algorithms (the underlying physics) would still need to be addressed, but it would be done so based on the testing performed by the quality engineers.  That is still safe and effective.   Not necessarily as efficient (the earlier you resolve issues the more efficient the process, typically, and code review typically takes place earlier in the process before verification/testing).  There are various ISO standards that are considered necessary and when taken in combination, sufficient for producing "safe and effective" software.  There are other related/harmonized standards (MDA, etc.) as well.

**Comment by SimonBiggs** [comment]
What about the following:

> Supporting Medical Physicists in making radiotherapy cancer treatment safer

We can then put the rest of the larger text from Stuart's comment above right below that?

**Comment by sjswerdloff** [comment]
I'm OK with that.
One can demonstrate that PyMedPhys is supporting a medical Physicists work, but it is the medical Physicist who makes the treatment safer, and that is how I interpret that top level byline.

I'm OK with having the broader set of interests in the secondary heading

**Comment by SimonBiggs** [comment]
> One can demonstrate that PyMedPhys is supporting a medical Physicists work, but it is the medical Physicist who makes the treatment safer, and that is how I interpret that top level byline.

Yup, that is how I indented it. Quite purposefully. Maybe it is better more explicit:

> Supporting Medical Physicists in their work of making radiotherapy cancer treatment safer
